content/5_Plat/Geo_Pri.mdx

```diff
@@ -403,6 +403,72 @@ int main() {
 
 - [My Templates](<https://github.com/bqi343/USACO/tree/master/Implementations/content/geometry%20(13)/Primitives>)
 
+## Angles
```

Comment:
can you reformat this module to follow [https://docs.google.com/document/d/1Dzj5uYNYNXe_E-_9-lSeX73654cGJk88PZ0kZCCmks0/edit](this module fmt)

Reply:
alright

Reply:
done